app/lib/pages/settings/network_interfaces_page.dart

```diff
@@ -43,9 +46,11 @@ class _NetworkInterfacesPageState extends State<NetworkInterfacesPage> {
         ? context.notifier(settingsProvider).setNetworkWhitelist
         : context.notifier(settingsProvider).setNetworkBlacklist;
     return Scaffold(
-      appBar: AppBar(
-        title: Text(t.networkInterfacesPage.title),
-      ),
+      appBar: Platform.isMacOS
+          ? appBarMac(t.networkInterfacesPage.title)
```

Comment:
As this logic is repeated on every appBar, it would be cleaner to make a dedicated "CustomAppbar" / "LocalSendAppbar" Widget that handles this logic.